optimum/exporters/onnx/convert.py

```diff
@@ -67,55 +68,44 @@ def check_dummy_inputs_are_allowed(
 
 
 def validate_models_outputs(
-    onnx_config: OnnxConfig,
-    reference_model: Union["PreTrainedModel", "TFPreTrainedModel"],
+    models: Dict[str, Tuple[Union["PreTrainedModel", "ModelMixin"], "OnnxConfig"]],
```

Comment:
We always do `models_for_validation = fn_get_models_from_config(reference_model, onnx_config)` so it could make sense to remove the arg

Reply:
For me it makes more sense to create `models_for_validation` outside the `export_models` and `validate_models_outputs`  functions: we only need to compute it once, currently the same dictionnary is generated two times. Also the arguments of `fn_get_models_from_config` could vary depending on the model type (for stable diffusion model, we only need the reference model). I agree that the argument name is misleading though, we can change it for `models_to_config` wdyt ?

Reply:
I would rather go with `models_and_onnx_configs` or something similar, the point being that it's a tuple not a dict.

Reply:
`models` is a dictionnary not a tuple, what about `models_to_onnx_configs` ?

Reply:
I mean you match the model "name" like `"encoder"` to a tuple looking like this `(model, onnx config)` right? Or the type annotation is wrong.

Reply:
Yes ! In your previous comment you were proposing to replace the name with `models_and_onnx_configs` to emphasize on the fact that it was a tuple no ?

Reply:
The values are tuples. The tuple both contains the model, and the onnx config. So `model_TO_onnx_configs` seems misleading to me since it sounds like values are dictionaries.

Reply:
I don't have a strong opinion on this so happy to change it with `models_and_onnx_configs` to close this conversation